src/Extend/SitesLinkedToMagentoStores.php

```diff
@@ -20,4 +22,38 @@ public function findByMageRunCode($code)
     {
         return collect($this->sites)->get($code);
     }
+
+    protected function getSavedSites()
+    {
+        return Cache::driver('array')->rememberForever('statamic_sites', function () {
+            $sites = [];
+            $stores = Rapidez::getStores();
+            $configModel = config('rapidez.models.config');
+
+            foreach ($stores as $store) {
+                if (config('rapidez.statamic.sites.' . $store['code'] . '.attributes.disabled')) {
+                    continue;
+                }
+
+                Rapidez::setStore($store['store_id']);
+
+                $locale = $configModel::getCachedByPath('general/locale/code');
+                $lang = explode('_', $locale)[0] ?? '';
+                $url = $configModel::getCachedByPath('web/secure/base_url');
```

Comment:
This happened to be some missing Rapidez config in the Magento repo. The .env of the Magento repo missed the Rapidez urls which led to the `web/secure/base_url` config being the Magento url instead of the Rapidez url.